turbo.json

```diff
@@ -7,7 +7,7 @@
       "outputs": ["dist/**"]
     },
     "dvc#build": {
-      "dependsOn": ["dvc-vscode-webview#build"],
+      "dependsOn": ["dvc-vscode-webview#build", "dvc-vscode-lsp#build"],
```

Comment:
Note to self: We will need to remember to move the built server next to/into the extension/dist folder before packaging.